### ISSUE TITLE
Feature request - Übernahme Dateiname in Meta-Titel

### ISSUE THREAD

**Issue by operatorone** [heimrichhannot/contao-filename-sanitizer-bundle#12]
Wäre es eine denkbare Option für die Erweiterung den Dateinamen, der ggf. schon sprechend formuliert wurde beim Hochladen in die Meta-Angaben unter Titel zu übernehmen, z. B. in die Standardsprache? 

Hintergrund ist der, dass es Backend-Benutzer gibt, die ihre Dateien schon entsprechend sinnvoll benennen, diese Benennung aber in keinster Weise als Dateiname verwendet werden kann und der extra Schritt, alle hochgeladenen Dateien, nochmals bearbeiten zu müssen wegfallen würde. Ob ein Redakteur nun viele oder wenige Dateien hochlädt, spielt eigentlich keine Rolle. Meines Erachtens, würde diese Funktion jedem Backend-Benutzer zugutekommen und man könnte sie optional in den Einstellungen machen, sodass die bisherige Funktionsweise die gleiche wäre und der Titel leer bliebe.

Viele Grüße
operatorone

**Comment by koertho** [comment]
@operatorone Hi, das klingt nach einer sinnvollen Ergänzung. Traust du dir einen PR zu? Wir sind momentan ziemlich ausgelastet.

**Comment by operatorone** [comment]
@koertho Hi, leider habe ich nicht die programmiertechnischen Fähigkeiten dafür, sonst würde ich da sofort einspringen.

**Comment by koertho** [comment]
@operatorone Was nicht ist, kann ja noch werden ;)

**Comment by operatorone** [comment]
Ja, das stimmt. Da ist die Lernkurve aber auch recht steil und die Zeit wie immer knapp ;-) Vielleicht kommt ihr ja irgendwann in nächster Zeit dazu. Das wäre echt ein cooles Feature für die Extension.
